src/sbus/connection/sbus_connection.c

```diff
@@ -435,6 +447,7 @@ sbus_connection_free_handler(struct tevent_context *ev,
                              struct timeval tv,
                              void *data)
 {
+    DEBUG(SSSDBG_TRACE_FUNC, "Actually releasing connection %p\n", data);
```

Comment:
Remove "Actually" ?

Reply:
I  meant to stress a difference with "Connection %p will be freed during next loop!\n", but I don't mind removing it.

Reply:
JFTR: the reason behind this additional debug is #6897

Reply:
I still don't think the word `Actually` adds any value here.

Reply:
Ok, removed.